README.md

```diff
@@ -82,6 +86,8 @@ export default routeResourceDetectorHOC(StudentInfo)
 | Name           | Type      | Default | Description                                                                                                                                                                                                                             |
 | -------------- | --------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `shouldDetectResourceForAllRoutes` | `boolean` | `true` | If `true`, the resources lies in all routes will be detected by default. |
+| `detectResourceInSequence` | `boolean` | `false` | If `true`, the resources will be detected in sequent, it will stop detecting resources when error occurs. See more in example. |
+| `deselectResourceBeforeRouteChanged` | `boolean` | `false` | If `true`, resources selected in the previous route will be deselected. See more in example. |
```

Comment:
If `true`, resource detector will invoke a deselect function provided by current route right before the route changes.